omise/payment_methods/true_money.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+if (defined('_PS_MODULE_DIR_')) {
+    require_once _PS_MODULE_DIR_ . 'omise/payment_methods/_offsite.php';
+}
+
+class OmisePaymentMethod_TrueMoney extends OmiseOffsitePaymentMethod
+{
+
+    const
+        NAME = 'TrueMoney',
+        PAYMENT_OPTION_NAME = 'omise-truemoney-payment',
+        DEFAULT_TITLE = 'True Money',
```

Comment:
According to Omise doc, I think the name that broadly used is `TrueMoney`, or `TrueMoney Wallet`

Reply:
Magento 1 and 2 use "True Money" - I copied it from there. I can change if it's a big deal

Reply:
Changed to "TrueMoney"